src/components/PopoverMenu.tsx

```diff
@@ -254,6 +259,7 @@ function PopoverMenu({
                             iconRight={item.iconRight}
                             shouldPutLeftPaddingWhenNoIcon={item.shouldPutLeftPaddingWhenNoIcon}
                             label={item.label}
+                            style={{backgroundColor: item.isSelected ? theme.activeComponentBG : undefined}}
```

Comment:
Looks like this addition led to the following issue:
- https://github.com/Expensify/App/issues/48543

Since given the new addition and the fact that in `SearchTypeMenuNarrow.tsx` we never pass the `isSelected` property. The issue was fixed by passing down `isSelected` which fixed the issue of the selected item highlight. More details in the issue [proposal](https://github.com/Expensify/App/issues/48543#issuecomment-2328889733).